src/library-authoring/edit-block/LibraryBlockPage.jsx

```diff
@@ -207,7 +218,10 @@ class LibraryBlockPage extends React.Component {
 
     return (
       <div className="library-block-wrapper">
-        <div className="wrapper-mast wrapper">
+        {/* todo: figure out what we want here, the header text always says "Component", and the 
+                  "back to library" button is redundant considering the header link with the
+                  library name goes back to the library */}
```

Comment:
Agreed.  I think it's fine to get rid of the redundancy.

Reply:
removed https://github.com/brian-smith-tcril/frontend-app-library-authoring/commit/9edd6f5f3a0936cd35b9b892055da724c537d603